csm/core/controllers/maintenance.py

```diff
@@ -66,6 +67,12 @@ async def post(self):
         action = self.request.match_info[const.ACTION]
         body = await self.request.json()
         body[const.ACTION] = action
+
+        #if hostname is obtained in request body then get the nodeid mapped to the hostname.
+        hostname_nodeid_map = Conf.get(const.CSM_GLOBAL_INDEX, f"{const.MAINTENANCE}")
```

Comment:
Could it be done in init ? Why does map need to be updated every time there is a request?

Reply:
Comment aaddressed, Please review